devtools/platform-descriptor-json/src/main/resources/templates/dockerfile-jvm.ftl

```diff
@@ -17,15 +17,18 @@
 FROM fabric8/java-alpine-openjdk8-jre:1.6.5
 ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
 ENV AB_ENABLED=jmx_exporter
-COPY ${build_dir}/lib/* /deployments/lib/
-COPY ${build_dir}/*-runner.jar /deployments/app.jar
-EXPOSE 8080
 
-# run with user 1001 and be prepared for be running in OpenShift too
+# Be prepared for be running in OpenShift too
```

Comment:
Minor complaint: `for be running` should probably be `for running`

Reply:
Fixed